coops-ui/src/mixins/entityFilter-mixin.ts

```diff
@@ -21,4 +21,24 @@ export default class EntityFilterMixin extends Vue {
   entityFilter (entityType: string): boolean {
     return this.entityType === entityType
   }
+
+  entityTypeToDisplay (): string {
+    switch (this.entityType) {
+      case 'CP':
+        return 'Cooperative'
```

Comment:
I don't see the requirement for this. Should the text fall back to "association" as before?